src/Psalm/Codebase.php

```diff
@@ -1159,4 +1160,31 @@ public function isTypeContainedByType(
     ): bool {
         return TypeAnalyzer::isContainedBy($this, $input_type, $container_type);
     }
+
+    /**
+     * @psalm-suppress PossiblyUnusedMethod
+     */
+    public function canTypeBeContainedByType(
```

Comment:
Maybe add a test for this and the method above, and add comments to explain to other potential plugin authors how they’re different (I realise I’m sort of asking you to do my work for me, but since you’re here...)

Reply:
Sure.

Reply:
Added docs.